client/layout.html

```diff
@@ -1,40 +1,15 @@
 <head>
-  <title>Make Lunch</title>
+  <script>
+    UPLOADCARE_PUBLIC_KEY = '15127b47da228b9b2d87'
```

Comment:
It should be, but I'm having a hard time making it so - the [Meteor Uploadcare package](https://github.com/MaximDubrovin/meteor-uploadcare) instructs to write the key in the <head> of the page as it needs to be available when the Uploadcare script is loaded, but template helpers can't be used in the <head> at the moment.